lib/trino-filesystem-s3/src/test/java/io/trino/filesystem/s3/TestS3Retries.java

```diff
@@ -111,7 +114,18 @@ public void testRetries()
                 .bucket(location.bucket())
                 .key(location.key())
                 .build();
-        S3Input input = new S3Input(location.location(), s3client, request);
+        S3Input input = new S3Input(
+                location.location(),
+                s3client,
+                request,
+                new S3Context(
+                        16 * 1000 * 1000,
```

Comment:
what does this magic value mean? 
Also the minimal value is calculated a little bit differently 
```
    private static final int MIN_PART_SIZE = 5 * 1024 * 1024; // S3 requirement
```
why the difference?